lib/placeorder.rb

```diff
@@ -0,0 +1,33 @@
+require "./lib/order.rb"
+#require "./lib/menu.rb"
+require "./twilio_text.rb"
+
+class PlaceOrder
+
+  def initialize(order_klass=Order, confirmation_klass=Confirmation)
+    @order_klass = order_klass.new
+    @confirmation_klass = confirmation_klass
+  end
+
+  def add_to_order(item)
+    @order_klass.select(item)
+  end
+
+  def checkout
+    price_check
+    message = "price does not match what you paid for please resart order process"
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)